### PR TITLE
Print a useful error if package.json is missing

### PR DESCRIPTION
**Pull request by nunospot**
Hello!

This fixes [#60](https://issuehunt.io/repos/12143193/issues/60)

If I understand it right we need to check if the package.json is missing or not, like @kirbysayshi said it on the issue. 
In this case I made a small condition to check this `['undefined', null].indexOf(config) > 0`.

Let me know if there is more about this issue than this.

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Referenced issues

This pull request has been submitted to:
- [#60: Fail with descriptive error if package.json is missing](https://issuehunt.io/repos/12143193/issues/60)
---

IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

### CODE REVIEW

**Comment by sindresorhus** [comment]
Please spend some more time on the code.

**Comment by nunospot** [comment]
Yeah, I saw the travis build fail, I will working on it. But beside that, did you see something wrong?

File: index.js [hunk 1]
```diff
@@ -11,6 +11,11 @@ module.exports = (grunt, options = {}) => {
 
 	let cwd = process.cwd();
 	let config = options.config || pkgUp.sync();
+
+	if (config === null) {
+		console.error('\u001B[41m%s\u001B[0m', 'ABORTED: package.json not found.\nPlease be sure to create a package.json and install all dependecies before run a grunt task.');
```
Comment (satazor) [inline index.js]: You should use `grunt.fail` to log errors and exit, see: https://gruntjs.com/api/grunt.fail

Comment (satazor) [reply index.js]: Moreover, by doing so, you don't need the error ansi codes at all.

Comment (nunospot) [reply index.js]: Thank you for the suggestion. I will work on that right away.

**Comment by sindresorhus** [comment]
Can you change the pull request title to be more descriptive? It should concisely explain what this PR fixes.

Also include `Fixes #123` with the issue number of the issue this pull request fixes in the pull request description so the issue is closed when this PR is merged.

File: index.js [hunk 2]
```diff
@@ -11,6 +11,11 @@ module.exports = (grunt, options = {}) => {
 
 	let cwd = process.cwd();
 	let config = options.config || pkgUp.sync();
+
+	if (config === null) {
+		grunt.fail.fatal('package.json not found.\nPlease make sure to create a package.json and install all dependecies before run a grunt task.');
```
Comment (sindresorhus) [inline index.js]: You have some typos.

```suggestion
		grunt.fail.fatal('package.json not found.\nPlease make sure to create a package.json and install all dependencies before you run Grunt.');
```

**Pull request merged**